src/BlazorUI/Bit.BlazorUI/Components/Inputs/TextField/BitTextField.scss

```
@@ -252,6 +252,27 @@
     }
+
+    .bit-tfl-ghw {
+        flex-grow: 1;
+    }
+
+    .bit-tfl-gho {
+        display: block;
+        overflow: auto;
+        align-items: initial;
+        white-space: pre-wrap;
+        word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)